packages/itwin/measure-tools/src/widgets/MeasurementWidget.tsx

```diff
@@ -39,11 +39,11 @@ export class MeasurementWidget extends WidgetControl {
     this._showPropertyAggregates = true;
     this._dataProvider = new SimplePropertyDataProvider();
     this._lastSelectedCount = MeasurementSelectionSet.global.measurements.length;
-    this._onSelectionChanged = this._onSelectionChanged.bind(this);
 
     if (options.iModelConnection) {
-      MeasurementSelectionSet.global.onChanged.addListener(this._onSelectionChanged);
-      MeasurementUIEvents.onMeasurementPropertiesChanged.addListener(this._onSelectionChanged);
+      MeasurementSelectionSet.global.onChanged.addListener(this._onSelectionChanged, this);
+      MeasurementUIEvents.onMeasurementPropertiesChanged.addListener(this._onSelectionChanged, this);
+      options.iModelConnection.onGlobalOriginChanged.addListener(this._onSelectionChanged, this);
```

Comment:
should this consideration make it to the UI2.0 widget:
https://github.com/iTwin/viewer-components-react/blob/master/packages/itwin/measure-tools/src/ui-2.0/MeasurementPropertyWidget.tsx